swatch-producer-aws/src/test/java/com/redhat/swatch/aws/service/AwsBillableUsageAggregateConsumerTest.java

```diff
@@ -20,8 +20,7 @@
  */
 package com.redhat.swatch.aws.service;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Fixed.  But I think we do use wildcards a lot in tests for static imports for JUnit, Mockito, and Hamcrest.  I don't think there's an official policy on this enforced by any tooling so it would be good to reach a team consensus on this.

```
% rg 'import static.*\*' -l | wc -l
83
```

So 83 classes with wildcards and nearly all of them are in `test` it looks like.  There are a handful in swatch-core and the main package.  We should probably either fix these or at least set the style linter to catch them.

Reply:
The [style guide](https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports) is unambiguous about forbidding wildcard imports.  So, now we just need to figure out why it's not enforcing that or if we want to as a team make an exception for `org.junit`, `org.mockito` stuff

Reply:
Haha, the formatter [doesn't](https://github.com/google/google-java-format/blob/20c526c79799c2f4a986bd2df621b022aa4c875e/core/src/test/java/com/google/googlejavaformat/java/ImportOrdererTest.java#L285) enforce it, that's why.

Reply:
Yes... I also tried a while ago to configure spotless to enforce this but it is not supported.
Maybe it's a good opportunity to start using ArchUnit to do this check.